### PR TITLE
finished adding changes for home page

### PR DESCRIPTION
**Pull request by Marumbo**

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
-        scrollRestoration: true,
-    },
-    images: {
-        domains: ["res.cloudinary.com", "images.unsplash.com", "cdn.sanity.io"],
-    }
+  reactStrictMode: true,
+  experimental: {
+    scrollRestoration: true,
+  },
+  images: {
```
Comment (liwoo) [inline next.config.js:7]: these should be in a `data` folder with different files such as `footer.ts`

File: src/app/page.tsx [hunk 2]
```diff
@@ -1,37 +1,41 @@
...
+    'CLA is a cell based Church that believes in the Bible and the power to change lives through a living relationship with Jesus Christ. Come join us!'
+  return (
+    <>
+      <Container className={'mt-8'}>
+        <NoticeWithImage notice={notice as any} />
+        <MediaHeroAction sermon={sermon} />
+        <div className="my-4 lg:my-20">
+          <PageTitle title={welcomeTitle} description={welcomeDescription} />
+        </div>
+        {/* @ts-expect-error Server Component */}
```
Comment (liwoo) [inline src/app/page.tsx:32]: why are we ignoring these items

File: src/components/index/Highlights.tsx [hunk 3]
```diff
@@ -1,80 +1,69 @@
...
+import { getCallToActions } from '@/sanity/home-page-data'
+import { truncateString } from '@/utils/helpers'
+import Link from 'next/link'
 
+export const Highlights = async () => {
+  const highlights = await getCallToActions()
+  return (
+    <div className="my-16 grid grid-cols-1 grid-rows-3 items-stretch gap-6 md:grid-cols-2 lg:grid-cols-3">
+      <div className="col-span-1 row-span-1">
+        {highlights[0] && (
```
Comment (liwoo) [inline src/components/index/Highlights.tsx:14]: I believe this should be `highlights.length > 0`

**Review by liwoo — CHANGES_REQUESTED** [review]

File: src/app/page.tsx (hunk 2 above)
Comment (Marumbo) [reply src/app/page.tsx:32]: still working on this error, from what im finding upgradign TS should have worked but no luck still

File: src/components/index/Highlights.tsx (hunk 3 above)
Comment (Marumbo) [reply src/components/index/Highlights.tsx:14]: the highlights are specific so it should be the first and the carry on

File: next.config.js (hunk 1 above)
Comment (Marumbo) [reply next.config.js:7]: moved these into a data folder

**Review by liwoo — APPROVED** [review]
looks ok... will fix some of the issues after merge

**Pull request merged**